<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLessonDialog</class>
 <widget class="QDialog" name="AddLessonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>650</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ثبت درس</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>461</width>
     <height>226</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_title">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_tedadVahed">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:white</string>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_type">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:white</string>
        </property>
        <item>
         <property name="text">
          <string>نظری</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>عمومی</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>عملی</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_17">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_field_2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white</string>
        </property>
        <property name="currentText">
         <string>مهندسی کامپیوتر - فناوری اطلاعات</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="maxVisibleItems">
         <number>20</number>
        </property>
        <item>
         <property name="text">
          <string>مهندسی کامپیوتر - فناوری اطلاعات</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>مهندسی کامپیوتر - نرم افزار</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>مهندسی عمران</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>مهندسی صنایع</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>مهندسی مکانیک</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>مهندسی معماری</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>مهندسی پزشکی</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>مهندسی شیمی</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>مهندسی مخابرات</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>مهندسی مواد</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>مهندسی هوافضا</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_title">
        <property name="font">
         <font>
          <family>B Koodak</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>نام :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_tedadVahed">
        <property name="font">
         <font>
          <family>B Koodak</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>تعداد واحد :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_type">
        <property name="font">
         <font>
          <family>B Koodak</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>نوع درس :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_18">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>B Koodak</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>رشته :</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>530</y>
     <width>941</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_back">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>B Nazanin</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(111, 111, 111);
color: white</string>
      </property>
      <property name="text">
       <string>برگشت</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_showAllCourses">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>B Nazanin</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(136, 182, 204)</string>
      </property>
      <property name="text">
       <string>نمایش تمام دروس</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_add">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>B Nazanin</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(118, 240, 179)</string>
      </property>
      <property name="text">
       <string>ثبت</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_title</tabstop>
  <tabstop>comboBox_tedadVahed</tabstop>
  <tabstop>comboBox_type</tabstop>
  <tabstop>comboBox_field_2</tabstop>
  <tabstop>pushButton_add</tabstop>
  <tabstop>pushButton_showAllCourses</tabstop>
  <tabstop>pushButton_back</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_back</sender>
   <signal>clicked()</signal>
   <receiver>AddLessonDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
